app/graphql/types/article_type.rb

```diff
@@ -17,6 +17,12 @@
   field :volume, !types.Int
   field :section, !Types::SectionType
 
+  field :comments, types[!Types::CommentType] do
+    resolve -> (obj, args, ctx) {
+      obj.comments.where.not(published_at: nil)
```

Comment:
Could you put this in a method? https://stackoverflow.com/questions/373731/override-activerecord-attribute-methods

Reply:
I'm changing it back to `published_comments`. At some points we will want the unpublished comments (for instance, listing unpublished comments to review for publication on the CMS).